src/printer/job.js

```diff
@@ -20,7 +20,8 @@ let counter = 0;
  * @param {PrintSpec} spec
  */
 export function createJob(spec) {
-  const frameState = getFrameState(spec);
+  const pixelSize = calculatePixelSize(spec);
```

Comment:
I think the name is confusing but maybe it's me. I understand it is the size of a pixel, but it is actually the size in pixel units.

Reply:
ok changed it to `sizeInPixel` ;-)